### PR TITLE
Alpha slidingform

### PR DESCRIPTION
**Pull request by Wagnerp**
* Some work on new features

### CODE REVIEW

File: Source/Krypton Toolkit/Krypton.Toolkit.Suite.Extended.Forms/Controls Toolkit/TransparentPanel.cs [hunk 1]
```diff
@@ -0,0 +1,59 @@
+﻿#region MIT License
+/*
+ * MIT License
+ *
+ * Copyright (c) 2017 - 2024 Krypton Suite
```
Comment (Smurf-IV) [inline Source/Krypton Toolkit/Krypton.Toolkit.Suite.Extended.Forms/Controls Toolkit/TransparentPanel.cs:5]: Just been added therefore start year is 2024

File: Source/Krypton Toolkit/Krypton.Toolkit.Suite.Extended.Forms/General/ControlExtensions.cs [hunk 2]
```diff
@@ -0,0 +1,256 @@
+﻿using System;
```
Comment (Smurf-IV) [inline Source/Krypton Toolkit/Krypton.Toolkit.Suite.Extended.Forms/General/ControlExtensions.cs:1]: Need license reference

File: Source/Krypton Toolkit/Krypton.Toolkit.Suite.Extended.Forms/General/Easing.cs [hunk 3]
```diff
@@ -0,0 +1,64 @@
+﻿using System;
```
Comment (Smurf-IV) [inline Source/Krypton Toolkit/Krypton.Toolkit.Suite.Extended.Forms/General/Easing.cs:1]: Lic ref

File: Source/Krypton Toolkit/Krypton.Toolkit.Suite.Extended.Forms/General/GraphicPathExtensions.cs [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+            var bottom = point.Y + size.Height;
+            var right = point.X + size.Width;
+
+            for (int i = 0; i < r.Length; i++)
+            {
+                r[i] = radius.Length > 0 ? radius[i % radius.Length] : 0f;
+                d[i] = 2f * r[i];
+            }
+
+            if (r[0] > 0f)
```
Comment (Smurf-IV) [inline Source/Krypton Toolkit/Krypton.Toolkit.Suite.Extended.Forms/General/GraphicPathExtensions.cs:61]: Apply braces to `if`s etc.

File: Source/Krypton Toolkit/Krypton.Toolkit.Suite.Extended.Forms/General/KryptonFormExtensions.cs [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+        public static Control Notify(this VisualKryptonFormExtended form, string message, int duration, double opacity, Color glowColor, Color backColor, Color foreColor)
+        {
+            var font = new Font(form.Font.FontFamily, form.Font.Size * 1.2f, form.Font.Unit);
+            var proposedSize = new Size(form.Width * 6 / 10, 0);
+            var size = TextRenderer.MeasureText(message, font, proposedSize, TextFormatFlags.WordBreak);
+            var panel = new TransparentPanel();
+            panel.Size = form.ClientRectangle.Size;
+            panel.Location = new Point(0, 0);
+            panel.Anchor = AnchorStyles.Left | AnchorStyles.Right | AnchorStyles.Top | AnchorStyles.Bottom;
+            panel.Paint += (sender, e) =>
```
Comment (Smurf-IV) [inline Source/Krypton Toolkit/Krypton.Toolkit.Suite.Extended.Forms/General/KryptonFormExtensions.cs:55]: Make this a separate functions as it it too big for inlining like this

**Review by Smurf-IV — CHANGES_REQUESTED** [review]

File: Source/Krypton Toolkit/Krypton.Toolkit.Suite.Extended.Forms/Controls Toolkit/TransparentPanel.cs (hunk 1 above)
Comment (Wagnerp) [reply Source/Krypton Toolkit/Krypton.Toolkit.Suite.Extended.Forms/Controls Toolkit/TransparentPanel.cs:5]: Revised year

File: Source/Krypton Toolkit/Krypton.Toolkit.Suite.Extended.Forms/General/ControlExtensions.cs (hunk 2 above)
Comment (Wagnerp) [reply Source/Krypton Toolkit/Krypton.Toolkit.Suite.Extended.Forms/General/ControlExtensions.cs:1]: Done

File: Source/Krypton Toolkit/Krypton.Toolkit.Suite.Extended.Forms/General/Easing.cs (hunk 3 above)
Comment (Wagnerp) [reply Source/Krypton Toolkit/Krypton.Toolkit.Suite.Extended.Forms/General/Easing.cs:1]: Done

File: Source/Krypton Toolkit/Krypton.Toolkit.Suite.Extended.Forms/General/GraphicPathExtensions.cs (hunk 4 above)
Comment (Wagnerp) [reply Source/Krypton Toolkit/Krypton.Toolkit.Suite.Extended.Forms/General/GraphicPathExtensions.cs:61]: Done

File: Source/Krypton Toolkit/Krypton.Toolkit.Suite.Extended.Forms/General/KryptonFormExtensions.cs (hunk 5 above)
Comment (Wagnerp) [reply Source/Krypton Toolkit/Krypton.Toolkit.Suite.Extended.Forms/General/KryptonFormExtensions.cs]: Done

**Pull request merged**